packages/flutter_tools/templates/package/README.md.tmpl

```diff
@@ -4,6 +4,11 @@
 
 ## Getting Started
 
-For help getting started with Flutter, view our online [documentation](https://flutter.io/).
-
-For help on editing package code, view the [documentation](https://flutter.io/developing-packages/).
+This project is a starting point for a
+[package](https://flutter.io/developing-packages/),
```

Comment:
I suggest we s/[package]/[Dart package]/ to be consistent with the the package type phrasing here:
https://flutter.io/docs/development/packages-and-plugins/developing-packages#package-types

Reply:
Do we want to link deeply to https://flutter.io/docs/development/packages-and-plugins/developing-packages#developing-dart-packages ?

Reply:
See other conversation.